src/Blocks/BlockIce.h

```diff
@@ -59,7 +59,7 @@ class cBlockIceHandler :
 			}
 		};
 
-		for (const auto Offset : Adjacents)
+		for (const auto & Offset : Adjacents)
```

Comment:
We're taking Vector3s by value though

Reply:
This isn't a function call, so the same arguments don't apply. I would be fine with by-value in this situation but it's not worth disabling an otherwise useful warning just to save one character.

Reply:
clang10 even adds a limit to how big the class is before it warns, clearly error here is bogus
codegen seems to be the same either way though, thankfully

Reply:
Yeah, I'm using clang 10 locally which is why I didn't pick it up in the first commit.